hugo/data/you_draw_it_karlsruhe.json

```diff
@@ -1,193 +1,40 @@
-    },
-    {
-      "year": 2011,
-      "value": 2629.205
-    },
-    {
-      "year": 2012,
-      "value": 2714.637
-    },
-    {
-      "year": 2013,
-      "value": 2658.536
-    },
-    {
-      "year": 2014,
-      "value": 2596.305
-    },
-    {
-      "year": 2015,
-      "value": 2558.452
-    },
-    {
-      "year": 2015,
-      "value": 2558.452
-    },
-    {
-      "year": 2016,
-      "value": 2461.506273333334
-    },
-    {
-      "year": 2017,
-      "value": 2364.5605466666666
-    },
-    {
-      "year": 2018,
-      "value": 2267.6148200000002
-    },
-    {
-      "year": 2019,
-      "value": 2170.6690933333334
-    },
-    {
-      "year": 2020,
-      "value": 2073.723366666667
-    },
-    {
-      "year": 2021,
-      "value": 1976.7776400000002
-    },
-    {
-      "year": 2022,
-      "value": 1879.8319133333334
-    },
-    {
-      "year": 2023,
-      "value": 1782.8861866666668
-    },
-    {
-      "year": 2024,
-      "value": 1685.9404600000003
-    },
-    {
-      "year": 2025,
-      "value": 1588.9947333333334
-    },
-    {
-      "year": 2026,
-      "value": 1492.0490066666669
-    },
-    {
-      "year": 2027,
-      "value": 1395.10328
-    },
-    {
-      "year": 2028,
-      "value": 1298.1575533333335
-    },
-    {
-      "year": 2029,
-      "value": 1201.2118266666669
-    },
-    {
-      "year": 2030,
-      "value": 1104.2661
-    },
-    {
-      "year": 2031,
-      "value": 1049.052795
-    },
-    {
-      "year": 2032,
-      "value": 993.8394900000001
-    },
-    {
-      "year": 2033,
-      "value": 938.626185
-    },
-    {
-      "year": 2034,
-      "value": 883.4128800000001
-    },
-    {
-      "year": 2035,
-      "value": 828.1995750000001
-    },
-    {
-      "year": 2036,
-      "value": 772.98627
-    },
-    {
-      "year": 2037,
-      "value": 717.7729650000001
-    },
-    {
-      "year": 2038,
-      "value": 662.55966
-    },
-    {
-      "year": 2039,
-      "value": 607.3463550000001
-    },
-    {
-      "year": 2040,
-      "value": 552.13305
-    },
-    {
-      "year": 2041,
-      "value": 496.919745
-    },
-    {
-      "year": 2042,
-      "value": 441.70644000000004
-    },
-    {
-      "year": 2043,
-      "value": 386.493135
-    },
-    {
-      "year": 2044,
-      "value": 331.27983000000006
-    },
-    {
-      "year": 2045,
-      "value": 276.066525
-    },
-    {
-      "year": 2046,
-      "value": 220.85321999999996
-    },
-    {
-      "year": 2047,
-      "value": 165.63991500000003
-    },
-    {
-      "year": 2048,
-      "value": 110.42660999999998
-    },
-    {
-      "year": 2049,
-      "value": 55.213305000000055
-    },
-    {
-      "year": 2050,
-      "value": 0
-    }
-  ]
+  "chart_id": "you-draw-it",
+  "chart": {
+    "heading": "Wie sollte sich der CO2-Ausstoß entwickeln?",
+    "lastPointShownAt": 2020,
+    "y_unit": "t. T.",
+    "data": [
+      {
+        "2005": 3075.886333333343
```

Comment:
Wie kommt dieser Wert in diese Datei?
In `karlsruhe.csv` steht er nicht.
In diesem PDF [Klimaschutzkonzept 2009](https://www.karlsruhe.de/b3/natur_und_umwelt/klimaschutz/klimakonzept/pdf_dateien/HF_sections/content/ZZmYk9MKZfe2na/Klimaschutzkonzept_Karlsruhe_2009.pdf) auf Seite 130 ist der Wert 5.948 für 2005 angegeben.